client/src/containers/ResetPassword/ResetPassword.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { FC } from "react";
+import { connect } from "react-redux";
+import axios from "axios";
+import * as Yup from "yup";
+import { withFormik, FormikProps, Form, Field } from "formik";
+import styled from "styled-components";
+import MadeWithLove from "../../components/MadeWithLove/MadeWithLove";
+import TextHelper from "../../components/TextHelper/TextHelper";
+import TextLink from "../../components/TextLink/TextLink";
+import TextField from "../../components/TextField/TextField";
+import Button from "../../components/Button/Button";
+import Container from "../../components/Container/Container";
+import { IconAvatarLock } from "../../components/AvatarIcon/AvatarIcon";
+import TextHeading from "../../components/TextHeading/TextHeading";
+import { addAlert, CanAddAlert } from "../../store/actions/alertActions";
+
+const StyledFormikForm = styled(Form)`
```

Comment:
We use it in every form so we can extract this to components

Reply:
I dont want to bother with this in this PR. But yes can be unify.